docs/intentReaderHandlers/SetApiResponseHandler.md

```diff
@@ -0,0 +1,92 @@
+# SetApiResponseHandler 
+
+## Overview
+
+SetApiResponseHandler is invoked when the task specified in the intent has the value "setApiResponse". This handler is used to assign the apiResponse value to an environment variable in FCA.
+
+It performs the following actions :
+1. Parse the input message received to get the apiResponse.
```

Comment:
Parses the input message received to get the `apiResponse`.

Reply:
updated for all handlers